### PR TITLE
neptune.new modules made a root modules

### PR DESCRIPTION
**Pull request by Raalsky**
## Before submitting checklist

- [x] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [x] Did you **asked the docs owner** to review all the updated user-facing interfaces?

### CODE REVIEW

File: src/neptune/new/__init__.py [hunk 1]
```diff
@@ -110,29 +103,10 @@ def _raise_legacy_client_expected(*args, **kwargs):
...
-    You may also want to check `get_last_run docs page`_.
-
-    .. _get_last_run docs page:
-       https://docs.neptune.ai/api/neptune/#get_last_run
-    """
-    last_run = Run.last_run
-    if last_run is None:
-        raise NeptuneUninitializedException()
-    return last_run
+warn_once(
```
Comment (Raalsky) [inline src/neptune/new/__init__.py:108]: @normandy7 That's for you :wink:

File: src/neptune/__init__.py [hunk 2]
```diff
@@ -13,10 +13,71 @@
...
+def get_last_run() -> Optional[Run]:
+    """Returns last created Run object.
+
+    Returns:
+        ``Run``: object last created by neptune global object.
+
+    Examples:
+        >>> import neptune
+
+        >>> # Crate a new tracked run
```
[the remark this answers is not in the document]
Comment (AleksanderWWW) [reply src/neptune/__init__.py:61]: Apart from that looks good to me, so will approve at once

[the remark this answers is not in the document]
Comment (Raalsky) [reply src/neptune/__init__.py]: Fixed + this function will be probably removed (and probably by you :P)

[the remark this answers is not in the document]
Comment (AleksanderWWW) [reply src/neptune/__init__.py]: XD

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ## [UNRELEASED] neptune-client 1.0.0
 
 ### Changes
+- Modules from `neptune.new` moved to `neptune` with compatibility imports and marked `neptune.new` as deprecated ([#1213](https://github.com/neptune-ai/neptune-client/pull/1213))
```
Comment (normandy7) [inline CHANGELOG.md:4]: Minor grammar fix.

```suggestion
- Moved modules from `neptune.new` to `neptune` with compatibility imports and marked `neptune.new` as deprecated ([#1213](https://github.com/neptune-ai/neptune-client/pull/1213))
```

**Review by normandy7 — APPROVED** [review]

**Review by AleksanderWWW — APPROVED** [review]

**Pull request merged**